CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changes by Version
 ==================
 
+0.29.0 (2021-06-25)
+-------------------
+* Bumped OpenTelemetry Collector to v0.29.0
```

Comment:
Could you add the following change to the changelog? 
https://github.com/open-telemetry/opentelemetry-operator/pull/313

And while you are at it, please add this to v0.28.0:
https://github.com/open-telemetry/opentelemetry-operator/pull/302
https://github.com/open-telemetry/opentelemetry-operator/pull/299
https://github.com/open-telemetry/opentelemetry-operator/pull/293

cc @VineethReddy02

Reply:
Sure I'll add it